dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div class="form-field">
+    <div data-test="dataPointToggle" class="form-field border-none vertical-middle">
+      <InputSwitch
+        data-test="dataPointToggleButton"
+        inputId="dataPointIsAvailableSwitch"
+        @click="this.isItActive = false"
+        v-model="isItActive"
+      />
+      <h5 data-test="dataPointToggleTitle" class="m-2">
+        {{ label }}
+      </h5>
+    </div>
+
+    <FormKit type="group" :name="name" v-if="isItActive">
+      <div>
```

Comment:
Is this `<div>` necessary?

Reply:
I think this has been fixed. The div now has a data-test attribute, which I believe is needed to wrap the element. FormKit doesn't render the attribute if you set it there.

Reply:
Ok then it makes sense. Closing this.